format/spec.md

```diff
@@ -585,13 +591,19 @@ The schema of a manifest file is a struct called `manifest_entry` with the follo
 | _optional_ | _optional_ | _optional_ | **`132  split_offsets`**          | `list<133: long>`                                                           | Split offsets for the data file. For example, all row group offsets in a Parquet file. Must be sorted ascending                                                                                                    |
 |            | _optional_ | _optional_ | **`135  equality_ids`**           | `list<136: int>`                                                            | Field ids used to determine row equality in equality delete files. Required when `content=2` and should be null otherwise. Fields with ids listed in this column must be present in the delete file                |
 | _optional_ | _optional_ | _optional_ | **`140  sort_order_id`**          | `int`                                                                       | ID representing sort order for this file [3].                                                                                                                                                                      |
-|            |            | _optional_ | **`142  first_row_id`**           | `long`                                                                      | The `_row_id` for the first row in the data file. See [First Row ID Inheritance](#first-row-id-inheritance)                                                                                                                 |
+|            |            | _optional_ | **`142  first_row_id`**           | `long`                                                                      | The `_row_id` for the first row in the data file. See [First Row ID Inheritance](#first-row-id-inheritance)                                                                                                        |
+|            | _optional_ | _optional_ | **`143  referenced_data_file`**   | `string`                                                                    | Fully qualified location (URI with FS scheme) of a data file that all deletes reference [4]                                                                                                                        |
```

Comment:
Do we want to allow V2 writers and readers to populate the referenced data file? I generally don't mind but we will have to continue to persist the bounds anyway to not silently break existing integrations that rely on reconstructing the referenced path from the bounds. Writing the field in addition to the bounds will make things only worse and require additional work to implement.

Reply:
I think we should allow it. We know that it is better to use `referenced_data_file` than to use the bounds, so it makes sense to allow it. People may not choose to use it if it is duplication.

Reply:
I agree. The problem, however, is that we won't switch to `referenced_data_file` anywhere in the library. Persisting both values won't make sense as it will increase the disk footprint and memory pressure. We can't stop writing the bounds too, as it may break external connectors like Trino.